frontend/www/js/omegaup/components/qualitynomination/DemotionPopupv2.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <omegaup-overlay-popup @dismiss="onHide">
+    <transition name="fade">
+      <form class="popup h-auto w-auto">
+        <template v-if="currentView == 'question'">
+          <div class="form-group">
+            <div class="font-weight-bold pb-4">
+              {{ T.reportProblemFormQuestion }}
+            </div>
+            <select
+              v-model="selectedReason"
+              class="control-label w-100"
+              name="selectedReason"
+            >
+              <option value="no-problem-statement">
+                {{ T.reportProblemFormNotAProblemStatement }}
+              </option>
+              <option value="poorly-described">
+                {{ T.reportProblemFormPoorlyDescribed }}
+              </option>
+              <option value="offensive">
+                {{ T.reportProblemFormOffensive }}
+              </option>
+              <option value="spam">
+                {{ T.reportProblemFormSpam }}
+              </option>
+              <option value="duplicate">
+                {{ T.reportProblemFormDuplicate }}
+              </option>
+              <option value="wrong-test-cases">
+                {{ T.reportProblemFormCases }}
+              </option>
+              <option value="other">
+                {{ T.reportProblemFormOtherReason }}
+              </option>
+            </select>
+          </div>
+          <div v-if="selectedReason == 'duplicate'" class="form-group">
+            <label class="control-label w-100">{{
+              T.reportProblemFormLinkToOriginalProblem
+            }}</label>
+            <input v-model="original" class="w-100" name="original" />
+          </div>
+          <div class="form-group">
+            <label class="control-label w-100">{{
+              T.reportProblemFormAdditionalComments
+            }}</label>
+            <textarea
+              v-model="rationale"
+              class="input-text w-100"
+              name="rationale"
+              type="text"
+            ></textarea>
+          </div>
+          <div class="text-right">
+            <button
+              class="col-md-4 btn btn-primary"
+              type="submit"
+              :disabled="!selectedReason || (!rationale &amp;&amp; selectedReason == 'other') || (!original &amp;&amp; selectedReason == 'duplicate')"
```

Comment:
```suggestion
              :disabled="!selectedReason || (!rationale && selectedReason == 'other') || (!original && selectedReason == 'duplicate')"
```

ya podemos usar `&` en las plantillas!

Reply:
Ah, cierto. Se cambia